pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -24,6 +24,7 @@
 , typst-lsp
 , typst-preview
 , autoPatchelfHook
+, xsel  # for ms-vsliveshare.vsliveshare
```

Comment:
```suggestion
```

Reply:
This issue is not resolved. No need to mark resolved.

Reply:
I did so such that the conversation would not get split into two locations. I think you might have missed [this](https://github.com/NixOS/nixpkgs/pull/301921#discussion_r1554294110). ^^